src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -4696,7 +4696,7 @@ int wm_vuldet_set_agents_info(agent_software **agents_software, update_node **up
         }
 
         if (!os_platform) {
```

Comment:
Here, we should do a `continue` to avoid a potential segfault when reaching the `strcasecmp`